### PR TITLE
Fix license type

### PR DESCRIPTION
**Pull request by sg-wbi**
This addresses the issue #689.

Container type for `Licenses` is changed to `SimpleNameSpace`.

```python
[ins] In [6]: from bigbio.utils.license import Licenses, License, CustomLicense

[ins] In [7]: custom = CustomLicense(text="abc")

[ins] In [8]: licens = Licenses.GPL_1p0

[ins] In [9]: type(custom), isinstance(custom, License)
Out[9]: (bigbio.utils.license.CustomLicense, True)

[ins] In [10]: type(licens), isinstance(licens, License)
Out[10]: (bigbio.utils.license.License, True)
```

### CODE REVIEW

File: tests/test_bigbio.py [hunk 1]
```diff
@@ -153,10 +163,21 @@ def test_metadata(self, module: ModuleType):
             metadata_attr = getattr(module, metadata_name)
 
             if metadata_name == "_LANGUAGES":
-                for lang in metadata_attr:
-                    if not isinstance(lang, metadata_type):
+
+                if not isinstance(metadata_attr, list):
```
Comment (sg-wbi) [inline tests/test_bigbio.py:167]: I have also updated the tests to make sure that  `_LANGUAGES` is not an empty list.

Comment (galtay) [reply tests/test_bigbio.py:167]: :+1:

**Review by galtay — APPROVED** [review]
